sechub-solution/docker/SecHub-Debian.dockerfile

```diff
@@ -63,6 +68,7 @@ RUN mkdir --parent "$BUILD_FOLDER" && \
     cd "sechub" && \
     "./buildExecutables" && \
     if [[ -n "$TAG" ]]; then git checkout tags/"$TAG" -b "$TAG"; fi && \
```

Comment:
git checkout after building makes no sense.
So maybe the tag/branch handling should happen before the build.